props/saber_BC_buttons.h

```diff
@@ -1039,6 +1040,7 @@ class SaberBCButtons : public virtual PropBase {
   bool battle_mode_ = false;
   bool max_vol_reached_ = false;
   bool min_vol_reached_ = false;
+  bool blade_detected_ = true;
```

Comment:
This creates two variables with the same name which is almost never a good idea.
Unless you have a terribly good reason for this, please solve it some other way.